rero_mef/agents/viaf/api.py

```
@@ -382,6 +388,50 @@ def update_online(self, dbcommit=False, reindex=False):
+            current_app.logger.warning(
+                f"VIAF redirect target not found or chained: "
+                f"{old_pid} -> {redirect_to_pid} | {msg}"
+            )
+            return None, Action.ERROR, redirect_info
+
+        # Delete old VIAF record (handles MEF cleanup)
+        self.delete(force=True, dbcommit=dbcommit, delindex=reindex)
+
+        # Create/update the target VIAF record
+        target_data.pop("redirect_from", None)
+        new_record, action = self.create_or_update(
```

Ruff (0.15.2)

[warning] Unpacked variable `action` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)